patches/BUNDLE-2554__set_payment_info_bug_fix__3.4.1.patch

```diff
@@ -0,0 +1,126 @@
+diff --git a/vendor/amzn/amazon-pay-module/Model/OrderInformationManagement.php b/vendor/amzn/amazon-pay-module/Model/OrderInformationManagement.php
+index 540b357..c18ef91 100644
+--- a/vendor/amzn/amazon-pay-module/Model/OrderInformationManagement.php
++++ b/vendor/amzn/amazon-pay-module/Model/OrderInformationManagement.php
+@@ -144,24 +144,11 @@ public function saveOrderInformation($amazonOrderReferenceId, $allowedConstraint
+             ];
+
+             $responseParser = $this->clientFactory->create($storeId)->setOrderReferenceDetails($data);
+-            try {
+-                $response       = $this->amazonSetOrderDetailsResponseFactory->create(
```

Comment:
Too much whitespace

Reply:
@BarnyShergold those lines are deleted. In patches need to pay attention to those which have `+` sign at the beginning, so only those line will exist after patch applying